### PR TITLE
New fields in scheduled csv reports.

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 

#### Description
Added there new ordered fields in Csv of CallCsvScheduler.

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/rest/brand/features/provider/billableCalls/getBillableCalls.feature [hunk 1]
```diff
@@ -19,6 +19,9 @@ Feature: Retrieve billable calls
               "callee": "+34633656565",
               "cost": null,
               "price": 1,
+              "carrierName": null,
```
Comment (mmadariaga) [inline web/rest/brand/features/provider/billableCalls/getBillableCalls.feature:22]: Add some data into fixtures

File: web/rest/client/features/provider/billableCall/getBillableCall.feature [hunk 2]
```diff
@@ -122,105 +132,105 @@ Feature: Retrieve billable calls
...
-017cc7c8-eb38-4bbd-9318-524a274f7091,"2019-01-01 09:01:31",0,+34633646464,+34633656565,1,,,,outbound,92,1
-017cc7c8-eb38-4bbd-9318-524a274f7092,"2019-01-01 09:01:32",0,+34633646464,+34633656565,1,,,,outbound,93,1
-017cc7c8-eb38-4bbd-9318-524a274f7093,"2019-01-01 09:01:33",0,+34633646464,+34633656565,1,,,,outbound,94,1
-017cc7c8-eb38-4bbd-9318-524a274f7094,"2019-01-01 09:01:34",0,+34633646464,+34633656565,1,,,,outbound,95,1
-017cc7c8-eb38-4bbd-9318-524a274f7095,"2019-01-01 09:01:35",0,+34633646464,+34633656565,1,,,,outbound,96,1
-017cc7c8-eb38-4bbd-9318-524a274f7096,"2019-01-01 09:01:36",0,+34633646464,+34633656565,1,,,,outbound,97,1
-017cc7c8-eb38-4bbd-9318-524a274f7097,"2019-01-01 09:01:37",0,+34633646464,+34633656565,1,,,,outbound,98,1
-017cc7c8-eb38-4bbd-9318-524a274f7098,"2019-01-01 09:01:38",0,+34633646464,+34633656565,1,,,,outbound,99,1
-017cc7c8-eb38-4bbd-9318-524a274f7099,"2019-01-01 09:01:39",0,+34633646464,+34633656565,1,,,,outbound,100,1
+callid,startTime,duration,caller,callee,price,destinationName,ratingPlanName,endpointType,endpointId,endpointName,direction,id,ddi
```
Comment (mmadariaga) [inline web/rest/client/features/provider/billableCall/getBillableCall.feature:135]: this is wrong

File: web/rest/client/features/provider/queue/getQueue.feature [hunk 3]
```diff
@@ -50,14 +50,15 @@ Feature: Retrieve queues
           "memberCallTimeout": 1,
           "strategy": "rrmemory",
           "weight": 5,
+          "preventMissedCalls": 1,
```
Comment (mmadariaga) [inline web/rest/client/features/provider/queue/getQueue.feature:53]: Changes in queue tests have nothing to do with this PR. Revert them

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**